src/SChain.ts

```diff
@@ -225,6 +225,70 @@ class SChain extends BaseChain {
         }
         return await transactions.send(this.web3, txData, opts);
     }
+
+    // todo: split - s2s transfers
+
+    async connectSchain(schainName: string, opts: TxOpts): Promise<any> {
+        const txData = this.contracts.tokenManagerLinker.methods.approve(schainName);
+        return await transactions.send(this.web3, txData, opts);
+    }
+
+    async transferToSchainERC20(
+        targetSchainName: string,
+        mainnetTokenAddress: string,
+        amount: string,
+        opts: TxOpts
+    ): Promise<any> {
+        const txData = this.contracts.tokenManagerERC20.methods.transferToSchainERC20(
+            targetSchainName,
+            mainnetTokenAddress,
+            amount
+        );
+        return await transactions.send(this.web3, txData, opts);
+    }
+
+    async transferToSchainERC721(
```

Comment:
missing transferToSchainERC721WithMeta cc @payvint

Reply:
we are working on it, under construction